e2e-tests/content-strings/error-messages.js

```diff
@@ -127,6 +127,10 @@ export const ERROR_MESSAGES = {
       [FIELD_IDS.INSURANCE.EXPORTER_BUSINESS.BROKER.TOWN]: {
         IS_EMPTY: 'Enter the town or city for your broker',
       },
+      [FIELD_IDS.INSURANCE.EXPORTER_BUSINESS.BROKER.EMAIL]: {
+        IS_EMPTY: 'Enter the email address for the broker in the correct format - for example name@email.com',
+        INCORRECT_FORMAT: 'Enter the email address for the broker in the correct format - for example name@email.com',
```

Comment:
Can we just have one decleration, `INCORRECT_FORMAT` instead of having duplication?

Also in other email fields we're using `name@example.com` instead of `name@email.com`. Can we check with design if we should be aligned here?